kong/pdk/init.lua

```diff
@@ -1,17 +1,13 @@
 ---
--- The Plugin Development Kit (or "PDK") is set of Lua functions and variables
+-- The Plugin Development Kit (PDK) is set of Lua functions and variables
 -- that can be used by plugins to implement their own logic. The PDK is a
 -- [Semantically Versioned](https://semver.org/) component, originally
--- released in Kong 0.14.0. The PDK will be guaranteed to be forward-compatible
--- from its 1.0.0 release and on.
---
--- As of this release, the PDK has not yet reached 1.0.0, but plugin authors
--- can already depend on it for a safe and reliable way of interacting with the
--- request, response, or the core components.
```

Comment:
Note that I removed this info because it looks like the PDK is now on 1.4. Is the new text accurate?